### ISSUE TITLE
TPC-DS tests

### ISSUE THREAD

**Issue by jcrist** [ibis-project/ibis#9447]
Ibis currently has a `TPC-H` test suite (found [here](https://github.com/ibis-project/ibis/tree/main/ibis/backends/tests/tpch)). To provide better coverage of a wider set of operations (and use for marketing/benchmarking purposes), it would be nice to also have tests for `TPC-DS` (see https://www.tpc.org/tpcds/).

**Comment by zhenzhongxu** [comment]
@jcrist looks like the TPC-H link is broken?

**Comment by cpcloud** [comment]
Yeah, I moved stuff around so it's become outdated. Probably not worth it to fix the link right now.
